crypto/codec/cmt.go

```diff
@@ -10,12 +10,12 @@ import (
 	bls12_381 "github.com/cosmos/cosmos-sdk/crypto/keys/bls12_381"
 	"github.com/cosmos/cosmos-sdk/crypto/keys/ed25519"
 	"github.com/cosmos/cosmos-sdk/crypto/keys/secp256k1"
-	cryptotypes "github.com/cosmos/cosmos-sdk/crypto/types"
+	sdkcrypto "github.com/cosmos/cosmos-sdk/crypto/types"
```

Comment:
do we need this change?

Reply:
IMO it helps to quickly differentiate between `cosmos/cosmos-sdk/crypto/types` (sdkcrypto) from `cosmos/crypto/types` (cosmoscrypto). The idea, at least for PubKey and PrivKey ifaces, is that in `cosmoscrypto` those ifaces are the "base" ones and then the types in sdkcrypto are `base _iface + some_iface` in the case of the sdk, the some_iface is proto.Message